content/docs/2.0/scalers/azure-log-analytics.md

```diff
@@ -0,0 +1,150 @@
+- `clientID`: appId from your Service Principal. Follow [this](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) link to create your Service Principal.
+- `clientSecret`: password from your Service Principal.
+- `workspaceID`: Your Log Analytics WorkspaceId. Follow [this](https://docs.microsoft.com/en-us/cli/azure/monitor/log-analytics/workspace?view=azure-cli-latest#az-monitor-log-analytics-workspace-list) link to get your Log Analytics WorkspaceId.
+- `query`: Log Analytics [kusto](https://docs.microsoft.com/en-us/azure/azure-monitor/log-query/get-started-queries) query, JSON escaped. You can use [this](https://www.freeformatter.com/json-escape.html) tool to convert your query from Log Analytics query editor to JSON escaped string, and then review YAML specific escapes.
+- `threshold`: An int64 value will be used as a threshold to calculate # of pods for scale target.
+
+**Threshold and query results:**
+
+It is important to design your query to return 1 table with 1 row. A good practice is to add "| limit 1" at the end of your query.
+Scaler will take value from:
+
+- 1st cell as Metrics Value.
+- 2d cell as Threshold (optional).
+
+You can define threshold in trigger metadata, it will be used if your query results only 1 cell, that will be interpreted as metric value. Be aware, even if you have defined threshold in metadata, it can be overwritten by your query.
+
+A data types of your query result should be: real, int or long. Other data types are not supported. Later, during runtime, your data will be converted to int64.
+
+Be careful with setting up "pollingInterval" and long running queries. Test your query before.
+
+Example query to get MetricValue and Threshold based on CPU usage and Limits, defined for the pod.
+```
+let AppName = "web";
+let ClusterName = "demo-cluster";
+let AvgDuration = ago(10m);
+let ThresholdCoefficient = 0.8;
+Perf
+| where InstanceName contains AppName
+| where InstanceName contains ClusterName
+| where CounterName == "cpuUsageNanoCores"
+| where TimeGenerated > AvgDuration
+| extend AppName = substring(InstanceName, indexof((InstanceName), "/", 0, -1, 10) + 1)
+| summarize MetricValue=round(avg(CounterValue)) by CounterName, AppName
+| join (Perf 
+        | where InstanceName contains AppName
+        | where InstanceName contains ClusterName
+        | where CounterName == "cpuLimitNanoCores"
+        | where TimeGenerated > AvgDuration
+        | extend AppName = substring(InstanceName, indexof((InstanceName), "/", 0, -1, 10) + 1)
+        | summarize arg_max(TimeGenerated, *) by AppName, CounterName
+        | project Limit = CounterValue, TimeGenerated, CounterPath, AppName)
+        on AppName
+| project MetricValue, Threshold = Limit * ThresholdCoefficient
+```
+
+You can convert your query in JSON escaped string using [this](https://www.freeformatter.com/json-escape.html) tool and check yaml specific escapes.
+
+**Scaler limitations (READ CAREFULLY):**

```

Comment:
Maybe move this to a `###` header after the `Authentication Parameters` section?

```suggestion
### Scaler Limitations
```